docs/gl_objects/wikis.rst

```diff
@@ -21,6 +24,10 @@ Get the list of wiki pages for a project::
 
     pages = project.wikis.list()
 
+Get the list of wiki pages for a group::
+
+    pages = group.wikis.list()
+
 Get a single wiki page::
 
     page = project.wikis.get(page_slug)
```

Comment:
Can we also get an example of doing a "get" of a group wiki page?

And an example of creating a group wiki page too.